views/play.erb

```diff
@@ -0,0 +1,14 @@
+<h1>Welcome to Rock Paper Scissors </h1>
+
+
+<h1>Player 1: <%= @name%></h1>
+<h2>Make your move </h2>
+<br>
+<form action="/play" method="post">
+  <input type="submit" name="sign" value="Rock"/>
+  <input type="submit" name="sign" value="Paper"/>
+  <input type="submit" name="sign" value="Scissors"/>
+</form> 
```

Comment:
I really like the options for RPS being buttons. However, once clicked, the `@name `variable disappears. Maybe you could include session data in your code so the @name variable is persisted throughout.